src/add/observable/combineLatest.ts

```diff
@@ -1,6 +1,4 @@
 import {Observable} from '../../Observable';
 import {combineLatestStatic} from '../../operator/combineLatest';
 
-Observable.combineLatest = combineLatestStatic;
-
-export var _void: void;
```

Comment:
I'll be reverting these to help the PR go through, but the good news is that empty .d.ts files won't be a problem for the consumer starting in tomorrow's builds (in other words we don't need the `_void: void` anymore).